static/js/components/dashboard/CourseAction_test.js

```diff
@@ -256,5 +259,29 @@ describe("CourseAction", () => {
       payButton.simulate("click")
       assert.equal(checkoutStub.calledWith(firstRun.course_id), true)
     })
+
+    it("enroll button redirects to order summary if full coupon and order fulfilled", () => {
```

Comment:
`will redirect to order summary by clicking enroll button if full coupon and order is fulfilled`